java2.js

```diff
@@ -0,0 +1,62 @@
+
+var values = {}
+
+function getValues () {
+  values.height = document.getElementById("userHeight").value;
+  values.character = document.getElementById("userChar").value;
+}
+
+function makeTree (values) {
+  if (values.height === "" || values.character === "") {
+    alert("Both fields must have a value.")
+   }
```

Comment:
I know that the requirement was just to print an alert (Again, if you feel like going above and beyond), but you can also prevent the rest of the function from actually running by returning the function early or wrapping it in the else statement to this if. (The code still tries to run when you submit and click ok from the alert)